tests/tests/test_mender_connect.py

```diff
@@ -33,12 +31,10 @@ def test_regular_protocol_commands(
     ):
         device = class_persistent_standard_setup_one_client_bootstrapped.device
 
-        if WRONG_BEHAVIOR:
-            # MEN-4273.
-            device.run(
-                """sed -i -e '2i"SkipVerify": true,' /etc/mender/mender-connect.conf"""
-            )
-            device.run("systemctl restart mender-connect")
+        device.run(
+            """sed -i -e '2i"SkipVerify": true,' /etc/mender/mender-connect.conf"""
+        )
+        device.run("systemctl restart mender-connect")
```

Comment:
This was under a `if WRONG_BEHAVIOR`, and hence this is what should not happen. There are more of these below.

Reply:
ack